### PR TITLE
Adding bi-direction streams

### PR DESCRIPTION
**Pull request by sushilks**

### CODE REVIEW

File: api/resourcediscovery/v1alpha2/resource_discovery.proto [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    DELETE = 2;
+  }
+
+  // REQUIRED. The operation to be performed in relation to the resource.
+  Operation operation = 5;
+
+  // this is the context that was send in the initials request call
+  string context = 6;
+}
+message BidirectionalStream {
```
Comment (venilnoronha) [inline api/resourcediscovery/v1alpha2/resource_discovery.proto:96]: nit: documentation missing.

File: api/resourcediscovery/v1alpha2/resource_discovery.proto [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    UPSERT = 1;
+
+    // Specifies a delete operation.
+    DELETE = 2;
+  }
+
+  // REQUIRED. The operation to be performed in relation to the resource.
+  Operation operation = 5;
+
+  // this is the context that was send in the initials request call
```
Comment (venilnoronha) [inline api/resourcediscovery/v1alpha2/resource_discovery.proto:93]: nit: s/send/sent
nit: s/initials/initial

File: api/resourcediscovery/v1alpha2/resource_discovery.proto [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  string context = 6;
+}
+message BidirectionalStream {
+  StreamRequest request = 1;
+  StreamResponse response = 2 ;
```
Comment (venilnoronha) [inline api/resourcediscovery/v1alpha2/resource_discovery.proto:98]: nit: extra space before semicolon.

File: api/resourcediscovery/v1alpha2/resource_discovery.proto [hunk 4]
```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 VMware, Inc. All Rights Reserved.
```
Comment (venilnoronha) [inline api/resourcediscovery/v1alpha2/resource_discovery.proto:1]: s/2019/2021

**Review by sergiopozoh — APPROVED** [review]

**Review by sergiopozoh — APPROVED** [review]

**Pull request merged**